docs/source/developing.rst

```diff
@@ -218,10 +218,31 @@ These two sets of variables can be identical, but they are separate because it i
 * ``RUNESTONE_HOST`` *d* - this is the canonical host name of the server.  It is used to generate links to the server.  It should be something like ``runestone.academy`` or ``runestone.academy:8000`` if you are running on a non-standard port.
 * ``LOAD_BALANCER_HOST`` *d* - this is the canonical host name of the server when you are running in production with several workers.  It is used to generate links to the server.  It should be something like ``runestone.academy`` or ``runestone.academy:8000`` if you are running on a non-standard port.  You would typically only need to set this or RUNESTONE_HOST.
 
-Variables that are important for the host side are probably best set in your login shell environment (such as a .bashrc file) But you can also set them in the .env file and as long as you have a RUNESTONE_PATH set commands like ``rsmanage`` and ``runestone`` will try to read and use those variables.  Variables that are important for the docker side are best set in the ``.env`` file.  The docker-compose file pulls
-When you are doing development you may want to set these in your login shell, But they can all be set in the ``.env`` file in the top level directory.  This file is read by docker-compose and the values are passed to the containers.  You can also set them in the ``docker-compose.yml`` file but that is not recommended.  The ``.env`` file is also used by the ``build.py`` script to set the environment variables for the docker-compose build.  As of this writing (June 2023) rsmanage does not know about the ``.env`` file so you will have to set them in your login shell if you want to use rsmanage.
-
-
+Variables that are important for the host side are probably best set in your
+login shell environment (such as a .bashrc file) But you can also set them in
+the ``.env`` file and as long as you have a RUNESTONE_PATH set commands like
+``rsmanage`` and ``runestone`` will try to read and use those variables.
```

Comment:
I'm a bit confused by this sentence compared to the disclaimer about `rsmanage` at line 232.

Reply:
To be clear, the thing I was confused about was at 232 where it says:

"As of this writing (June 2023) rsmanage does not know about the ``.env`` file so you will have to
set them in your login shell if you want to use rsmanage."